packages/colorpicker/lib/src/state/color_state.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:flutter/material.dart';
+import 'package:riverpod_annotation/riverpod_annotation.dart';
+
+part 'color_state.g.dart';
+
+@riverpod
+class ColorState extends _$ColorState {
+  @override
+  Color build() {
+    return Colors.blue;
```

Comment:
Shouldn't this use your declared colours in the constants folder?
I think we should just use the first one from the list there, so if I change the color there it automatically changes here